### ISSUE TITLE
Styling issue on Android 4.3

### ISSUE THREAD

**Issue by darth-willy** [humhub/humhub#1950]
Hello all,

When trying to display a user's dashboard (Humhub 1.1.0) on Android (version 4.3, GenyMotion emulator), the header is shifted way over to the right, and none of its (still viewable) options can be selected:

![device-2016-08-21-185243](https://cloud.githubusercontent.com/assets/11859376/17839996/7f87568c-67d0-11e6-9c49-ecc5ab69ebb7.png)

However, the same issue does not occur when displaying their profile or any other page, as far as I can tell. Any ideas? 

Thanks,

William

**Comment by cebe** [comment]
For me, in Android stock Browser, the top nav does not show up at all:

![screenshot_2016-08-22-09-52-48](https://cloud.githubusercontent.com/assets/189796/17847820/6602f51a-6850-11e6-8d2b-4299396e0675.png)

Works fine in Firefox though:

![screenshot_2016-08-22-09-55-34](https://cloud.githubusercontent.com/assets/189796/17847819/65ecb976-6850-11e6-8244-04dbe1f0c4a5.png)

**Comment by williamvanwoensel** [comment]
I've been playing around with it a bit, and replacing "position: fixed" by "position: absolute" for the topbar CSS class works, although that clearly means the topbar will scroll together with the rest of the content.

However, removing the "max-height: 200px" for screens up to 768px width ("see @media screen and (max-width:768px)" part) seems to work for me: the bar remains fixed and the issue no longer occurs.

@cebe , does this do the trick for you too?

**Comment by FiSHYsolutions** [comment]
I had the same issue like @cebe, running HumHub v1.1.2, with Android 4.2.2 and Lightning 4.4.1 (and also Androids Stock Browser v4.2.2-2015081400), but **not** with a very narrow Firefox 53.0.2 (Desktop Ubuntu 16.04 LTS). `div#topbar-first` and `div#topbar-second` are available on my mobile device until the overview-stream (dashboard) is loaded, then both bars are disappeared and it's impossible to use the menu-items.

I modified 'theme.css' but i didn't replaced `fixed` with `absolute` in the `.topbar`-class, as suggested by @darth-willy. I added instead `.topbar{position:absolute !important;}` in the `@media screen and (max-width:920px){...}`-section.

now i have the default behaviour on screens wider than 920px (where the sidebar-container comes up on the right of the content-container). everything else uses a not-scrollable absolute-positioned topbar. i have no idea why this issue happens, but that fixed it for me (until now).
